ec2uploadimg

```diff
@@ -703,15 +701,25 @@ def get_vpc_subnet_id(args, config, region, setup,  logger):
                 'subnet_id_%s' % region,
                 '--vpc-subnet-id'
             )
+            return _vpc_subnet_id_with_logger(vpc_subnet_id, logger)
+        except Exception:
+            pass
+        try:
+            # falls back to setup.create_vpc_subnet()
+            vpc_subnet_id = setup.create_vpc_subnet()
+            return _vpc_subnet_id_with_logger(vpc_subnet_id, logger)
         except Exception:
             msg = 'Not using a subnet-id, none given on the '
-            msg += 'command line and none found in config for '
-            msg += '"subnet_id_%s" value' % region
+            msg += 'command line, none found in config for '
+            msg += '"subnet_id_%s" value ' % region
+            msg += 'and unable to create a VPC Subnet'
```

Comment:
jfyi, you can set multiline strings in python with, imo, a cleaner syntax:
```
msg = (
     'line 1'
     'line 2'
     ...
)
```

Additionally the `f-string` notation is also more readable
```
msg = f'subnet_id_{region} value'
```

Good job with this PR! :mechanical_arm:

Reply:
Nice, thanks, Andrés!